src/scripts/extensions/styledFrameFactory.ts

```diff
@@ -43,6 +43,7 @@ export class StyledFrameFactory {
 		iframe.style.position = "fixed";
 		iframe.style.transition = "initial";
 		iframe.style.zIndex = "2147483647";
+		(iframe.style as any).colorScheme = "light";
```

Comment:
⁠how about declaring upfront that we only support light mode?

:root {
  color-scheme: light; // or only light
}

in our styles, hopefully it will not be overridden then?